manifests/o/Orange-OpenSource/Hurl/1.5.0/Orange-OpenSource.Hurl.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Orange-OpenSource.Hurl
+PackageVersion: 1.5.0
+PackageLocale: en-US
+Publisher: Orange-OpenSource
+PackageName: Hurl

```

Comment:
```suggestion
PackageName: Hurl - Command line tool that runs HTTP requests defined in a simple plain text format.
```

Reply:
This needs to match Control Panel.

Reply:
I have opened a PR for the `hurl` repository to fix this: https://github.com/Orange-OpenSource/hurl/pull/406.

In the next version, the `PackageName` should be changed back to `Hurl`.

Reply:
Won't this still match anyways, since the CLI looks for `Hurl*` ?

Reply:
Hi @ItzLevvie :)

I've merged your https://github.com/Orange-OpenSource/hurl/pull/406, PackageName and DisplayName will match on the next release (1.6.0).